### PR TITLE
Adds myself to the Maptainers as a codeowner

### PR DESCRIPTION
**Pull request by Jolly-66**
I also reorganized the list a bit- I hope you don't mind!

Also don't know if this'll work.

### CODE REVIEW

**Comment by Zonespace27** [comment]
DNM until you get invited to the repo/org so the code owners file doesn't complain

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -24,10 +24,13 @@
...
 # Zonepace
 /modular_skyrat/master_files/code/modules/client/preferences/headshot.dm @Zonespace27
 /modular_skyrat/modules/contractor/ @Zonespace27
 /modular_skyrat/modules/gun_cargo/ @Zonespace27
+
+# Multiple Owners
+
+# Maptainers
+/_maps/ @Jolly-66
+/_maps/ @KathrinBailey
```
Comment (Fikou) [inline .github/CODEOWNERS]: thats not how you do this

Comment (Jolly-66) [reply .github/CODEOWNERS]: What do you mean? What are you specifically commenting on?

image

**Review by GoldenAlpharex — APPROVED** [review]
Bottomtainer ❤️

**Pull request merged**